index.html

```diff
@@ -138,5 +138,48 @@ <h1 class="me">About</h1>
     </div>
     </article>
```

Comment:
[OPTIONAL]
- [ ] Kindly always make sure the previous work (`milestone`) is included in your newly opened pull request